x-pack/legacy/plugins/epm/public/data.ts

```diff
@@ -89,3 +91,16 @@ export async function installDatasource(datasource: DatasourcePayload): Promise<
   const body = JSON.stringify(datasource);
   return _fetch(path, { body, method: 'POST' });
 }
+
+// This should come from the shared Ingest types
```

Comment:
Could we add a `TODO` here to make sure we find it again for cleanup?

Reply:
👍 Added in bae5244393cc449375d9bbf902655a47fdae1f3e